core/core-configuration-layer.el

```diff
@@ -689,7 +692,10 @@ path."
               (configuration-layer//get-latest-package-version-string
                pkg-name)))
       ;; (message "%s: %s > %s ?" pkg-name cur-version new-version)
-      (version< cur-version new-version))))
+      (if new-version
+          (version< cur-version new-version)
+        (cl-pushnew pkg-name configuration-layer--skipped-packages :test #'eq)
```

Comment:
Ok but in this particular case you use `:test ' eq` so I wonder what is the benefit over regular `add-to-list` which does basically this.